taf/messages.py

```diff
@@ -7,6 +7,7 @@
         "add-target": "Add new target {target_name}",
         "remove-target": "Remove target {target_name}",
         "remove-from-delegated-paths": "Remove {target_name} from delegated paths",
+        "register-targets": "Sign targets and metadata"
```

Comment:
By registering targets, I usually mean update and sign metadata in case when targets content was changed. It does not matter if the targets already exist on disk or not. Can you think of a less confusing name?

Reply:
Would it be more precise?

```
"update-targets": "Update targets and sign metadata"
```